.github/no-response.yaml

```diff
@@ -1,18 +0,0 @@
-#

```

Comment:
@mgresko did you intent to remove this file?

Reply:
Yes. Didn’t think we wanted to auto close merge requests right now.

Reply:
Makes sense